frontend/src/components/Bill/Bill.jsx

```diff
@@ -162,21 +174,28 @@ export const Bill = () => {
             isSenate={isSenate}
           />
         )}
-        <div className="important">
-          <h4>Why is this important? Why should this bill pass?</h4>
+      </div>
+      
+      <div className="action">
+        <h2>Take Action!</h2>
+        <LocForm sponsorNames={getSponsorNames(bill)}/>
+        <div className="script">
+          <h2>Script when calling a Legislator</h2>
+          <p>
+            Hi, my name is [Your Name] and I'm a constituent from {window.sessionStorage.getItem('placeName')}. I'm calling to urge [Legislator's Name] to support {printNo}.
```

Comment:
```suggestion
            Hi, my name is [Your Name] and I'm a constituent from {window.sessionStorage.getItem('placeName') || '[Borough]'}. I'm calling to urge [Legislator's Name] to support {printNo}.
```


This shows nothing if the user hasn't filled it out, so I'm suggesting a default value